pkg/actions/commands.go

```diff
@@ -217,6 +217,63 @@ func Commands() {
 						return nil
 					},
 				},
+				{
+					Name:  "link",
+					Usage: "Manage project links",
+					Subcommands: []cli.Command{
+						{
+							Name:    "list",
+							Aliases: []string{"ls"},
+							Usage:   "Lists all the links for a project",
+							Flags: []cli.Flag{
+								cli.StringFlag{Name: "id,i", Usage: "Project ID", Required: true},
```

Comment:
big nit: spaces after commas in the flags (like the other commands)

Reply:
I've done this but fyi some others don't have spaces